kernel/shared/src/main/scala/cats/effect/kernel/Resource.scala

```diff
@@ -1279,6 +1302,7 @@ private[effect] trait ResourceHOInstances3 extends ResourceHOInstances4 {
 }
 
 private[effect] trait ResourceHOInstances4 extends ResourceHOInstances5 {
+  @deprecated("Use MonadCancelThrow instances", "3.6.0")
```

Comment:
Maybe a message like this can be more helpful to the user?

```suggestion
  @deprecated("Bring an implicit MonadCancelThrow[F] into scope to get the fixed Resource instance", "3.6.0")
```